### ISSUE TITLE
Exclude ignored pages with `ignorePrefix`

### ISSUE THREAD

**Issue by WolfgangDrescher** [harlan-zw/nuxt-simple-sitemap#30]
### Clear and concise description of the problem

Currently it is possible in nuxt to ignore pages with the `ignorePrefix` option in `nuxt.config.js` (https://nuxt.com/docs/api/configuration/nuxt-config#ignoreprefix). It would be nice if this module would exclude those pages automatically from the sitemap.

### Suggested solution

An easy solution would be to find glob pattern that match the `ignorePrefix`.  Something like this (does not work for me):

```js
export default defineNuxtConfig({
  sitemap: {
    exclude: [
      '/**/-*',
      '/-*',
    ],
  },
});
```

### Alternative

Maybe `.nuxtignore` could also be relevant for this: https://nuxt.com/docs/guide/directory-structure/nuxtignore

**Comment by harlan-zw** [comment]
Hi @WolfgangDrescher 

Thanks for the issue. I think the `ignorePrefix` in Nuxt isn't actually working as it should. I've made a PR here: https://github.com/nuxt/nuxt/pull/20202

Outside of that, if the issue is still present would you be able to provide some reproduction example?  The page scanning should already honor the Nuxt logic in ignoring pages.

**Comment by harlan-zw** [comment]
Okay found a solution that's available in v2.4.10

**Comment by WolfgangDrescher** [comment]
Just tested this in my project and it works fine. Thank your for implementing it that fast.
